integration/network/macvlan/macvlan_test.go

```diff
@@ -640,3 +641,35 @@ func TestPointToPoint(t *testing.T) {
 	assert.Check(t, is.Equal(res.Stderr.Len(), 0))
 	assert.Check(t, is.Contains(res.Stdout.String(), "1 packets transmitted, 1 packets received"))
 }
+
+func TestEndpointWithCustomIfname(t *testing.T) {
+	// verify the driver automatically provisions the 802.1q link (dm-dummy0.60)
+	skip.If(t, testEnv.IsRemoteDaemon)
+	skip.If(t, testEnv.IsRootless, "rootless mode has different view of network")
+
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	master := "dm-dummy0"
```

Comment:
The comment at the top should probably go here, as this is where that part of the test happens?

```suggestion
	// verify the driver automatically provisions the 802.1q link (dm-dummy0.60)
	master := "dm-dummy0"
```

Reply:
Also for fixed values, I started to use `const` more; I think it sometimes help glancing over a test to identify "dynamic" values (may change during the test) and fixtures;

```suggestion
	// verify the driver automatically provisions the 802.1q link (dm-dummy0.60)
	const master = "dm-dummy0"
```

Reply:
Ah, yeah I had these changes parked yesterday while I was trying to fix unobvious concurrency issues in the osl package. I should have put this PR back in draft mode.

I removed these comments as they don't say anything useful, and are even wrong. It was copy/pasted from other tests -- we should probably remove them in those other tests too.